sale_timesheet_invoice_description/models/sale.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models, _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    timesheet_invoice_description = fields.Selection(
+        '_get_timesheet_invoice_description', default='111')
```

Comment:
We have said that the default should be put at company level and accessible from the Sales > Settings menu.